packages/admin-panel/src/pages/resources/ResourcePage.jsx

```diff
@@ -128,13 +171,18 @@ ResourcePage.propTypes = {
   defaultSorting: PropTypes.array,
   defaultFilters: PropTypes.array,
   editorConfig: PropTypes.object,
+  detailsView: PropTypes.object,
+  parent: PropTypes.object,
+  displayProperty: PropTypes.string,
+  getIsLink: PropTypes.func,
+  getDisplayValue: PropTypes.func,
+  getLink: PropTypes.func,
```

Comment:
```suggestion
  getDetailsViewLink: PropTypes.func,
```
Similar reasoning to above